tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -21,10 +21,17 @@ DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
 container_image="${SHIM_V2_CONTAINER_BUILDER:-$(get_shim_v2_image_name)}"
 
+EXTRA_OPTS="${EXTRA_OPTS:-""}"
+
+http_proxy="${http_proxy:-}"
+https_proxy="${https_proxy:-}"
```

Comment:
better keep it consistent with other `build.sh` scripts and leave the proxy settings out?

Reply:
Thanks @mythi , fixed now.